src/ui/CategoryFacet/CategoryFacet.ts

```diff
@@ -269,6 +271,37 @@ export class CategoryFacet extends Component implements IAutoLayoutAdjustableIns
      * > Using value captions will disable alphabetical sorts (see the [availableSorts]{@link Facet.options.availableSorts} option).
      */
     valueCaption: ComponentOptions.buildJsonOption<IStringMap<string>>({ defaultValue: {} }),
+    /**
```

Comment:
Suggestion:

> The [`id`](@link DynamicFacet.options.id) of another dynamic facet in which at least one value must be selected in order for the dependent dynamic facet to be visible.
>
> **Default:** `undefined`, and the dynamic facet does not depend on any other dynamic facet to be displayed.

Reply:
Should I keep the example or delete it?

Reply:
also, why is it pointing to dynamic facet instead of category facet?

Reply:
Sorry, it should definitely point to CategoryFacet!

I would remove the example. I think it's fairly straightforward to use and I'm on a bit of a rampage to simplify the reference documentation ;)

Reply:
So the first part of the description should read:

> The [id]{@link Facet.options.id} of another facet in which at least (...) in order for the dependent category facet (...)

And the last part:

> **Default:** `undefined`, and the category facet does not depend on any other facet to be displayed.